docs/src/components/Story.tsx

```diff
@@ -27,14 +27,17 @@ export const Story = ({ path, ...props }: StoryProps) => {
         <BrowserOnly>
             {() => {
                 // https://docusaurus.io/docs/docusaurus-core/#browseronly
-                // eslint-disable-next-line @typescript-eslint/no-var-requires
+
+                // eslint-disable-next-line @typescript-eslint/no-require-imports
                 const { MuiThemeProvider } = require("@comet/admin");
 
                 return (
                     <IntlProvider locale="en">
                         <MuiThemeProvider theme={theme}>
                             <CodeBlock
                                 language="tsx"
+                                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                                // @ts-ignore
                                 live
```

Comment:
i am unsure if we need the live  prop on Codeblock Component, its not available in the types, and i could not find them in documentation.